src/EditorFeatures/CSharpTest/CodeLens/CSharpCodeLensTests.cs

```diff
@@ -215,35 +215,38 @@ public class A
             await RunMethodReferenceTest(input);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeLens)]
-        public async Task TestFullyQualifiedName()
+        [Theory, Trait(Traits.Feature, Traits.Features.CodeLens)]
+        [InlineData("class", "class", "class")]
+        [InlineData("record class", "record class", "record class")]
+        [InlineData("record struct", "record struct", "record struct")]
```

Comment:
i don't get the need for triplicate (since they're all the same).  why not just pass in one, and reference hte paramter three times?

Reply:
Initially I thought I'd include some mix&match combinations, but it wasn't necessary to hit the cases I wanted. I'll simplify

---
In reply to: [609207610](https://github.com/dotnet/roslyn/pull/52464#discussion_r609207610) [](ancestors = 609207610)